Rdmp.Core/DataLoad/Modules/DataFlowSources/DelimitedFlatFileDataFlowSource.cs

```diff
@@ -391,22 +391,21 @@ protected void OpenFile(FileInfo fileToLoad)
             {
                 Delimiter = Separator,
                 HasHeaderRecord = string.IsNullOrWhiteSpace(ForceHeaders),
-                ShouldSkipRecord = ShouldSkipRecord
+                ShouldSkipRecord = ShouldSkipRecord,
+                IgnoreBlankLines =  IgnoreBlankLines,
+                Mode = IgnoreQuotes ? CsvMode.Escape : CsvMode.RFC4180,
```

Comment:
Ignore Quotes might be CsvMode.NoEscape ?

Reply:
Can you make sure that you have the test `Test_IgnoreQuotes` in your working copy and that it passes?